UefiPayloadPkg/UefiPayloadPkg.dsc

```diff
@@ -570,6 +570,12 @@
   SecurityPkg/VariableAuthenticated/SecureBootDefaultKeysDxe/SecureBootDefaultKeysDxe.inf
 !endif
 
+!include DasharoModulePkg/DasharoModuleComponents.dsc.inc
+!include DasharoModulePkg/Include/UserAuthFeature.dsc

```

Comment:
I don't like this actually.

UserAuthFeature.dsc contains `[LibraryClasses]` and `[Components.X64]` which can't be included here.
Same with `DasharoModuleComponents.dsc.inc`, it includes libraries and modules that are duplicated in the UefiPayloadPkg.dsc. 

Include the Dasharo password modules directly here without including DSC.